how-to/disable-mods/multimc.md

```diff
@@ -0,0 +1,106 @@
+---
+hidden: true
+---
+
+# MultiMC
+
+{% hint style="danger" %}
+Do NOT disable "API" or "library" mods, because the other mods need them.
+{% endhint %}
+
+Instructions vary depending on whether you had installed FO on MultiMC with the easier installation or with automatic updates.
+
+<!-- TODO: instructions for manual installation? -->
```

Comment:
yes ofc manual instructions are necessary
the whole reason I didn't write MMC AU instructions there explicitly was that it is incredibly clumsy